### PR TITLE
Householding

### PR DESCRIPTION
**Pull request by charlydelta**
Just a few chore-changes 

**TL;DR**

- Don't require `grunt` and `bower` to be global dependencies
- Add `make watch` to substitute `grunt watch`
- Eased instructions in the README
- Ignore node debug files

**Explanations**

Hey, I just wanted to help out and had issues initially setting up the project\*, so I made the setup more comfortable. A few thoughts:

- You are already primary using `make` for the tasking, so you can basically hide everything there. It's super confusing for new people if they have to deal with that much tasking and packaging stuff. I mean, you got package dependencies, bower, grunt, npm, make, ...
- Using the `scripts` part of the `package.json` is very comfortable to define scripts and tasks, since it adds `node_modules/bin` to its path. No need to install anything globally
- By not having global node deps, you can specifically set the versions so no dev will have issues with weird version incompatibilities 

**\* The issues I had**
The `bower` command was not available, so it crashed when running `make setup`, even though I could run `bower` in the very same terminal without `make` in between.

### CODE REVIEW

**Comment by charlydelta** [comment]
Shouldn't have done a rebase, this MR is a mess as well... I am so sorry, I will fix it up here now as well.
